github-profiles/adjsvar-8685/README.md

```diff
@@ -0,0 +1,27 @@
+# Adrian Silva.
+
+Hi there!, I am a self-taught software development student, focused mostly in the web area. I have worked with technologies such as **HTML, CSS, Bootstrap, JavaScript (ES6), Python, SQL, and React.** Although I love the frontend, my goal is to be proficient in the backend technologies as well.
+
+Actually i'm following the Javascript Fullstack Bootcamp from Guillermo Rodas and here is my homework:
+
+5 Most used commands:
+
+| Command | Description                         |
```

Comment:
Everything in the file is awesome. But fix the alignment in the table header

Reply:
Thanks dan, i think it's done!